src/ensembl/src/content/app/species-selector/components/species-selector-app-bar/SpeciesSelectorAppBar.tsx

```diff
@@ -54,6 +55,9 @@ const SelectedSpeciesList = (props: Props) => {
           onRemove={props.onSpeciesDelete}
         />
       ))}
+      <div className={styles.genomeBrowserLinkContainer}>
+        <Link to="/app/browser">View in Genome Browser</Link>
```

Comment:
urlFor shared helper is now available on dev:

import * as urlFor from 'src/shared/helpers/urlHelper';
urlFor.browser()

Reply:
Thanks @msmanoj. I am already using it in the bootstrap branch.

Reply:
Rebased onto the fresh dev and added the urlHelper here.

_(I think, in the near future we will need to update this helper method to be able to generate links to specific genome, region and focus object)_